distrobox-init

```diff
@@ -195,6 +204,15 @@ mount_bind() (
 
 	return 0
 )
+
+if [ -n "${pre_init_hook}" ]; then
+printf "distrobox: Executing pre-init hooks...\n"
```

Comment:
https://github.com/89luca89/distrobox/runs/6434173848?check_suite_focus=true

line 209 to 212 should be indented or lint won't pass